### PR TITLE
[MOD-2322] Add mapclick event to MapView in Ti.Map Module

### PR DESCRIPTION
**Pull request by bridgepoint**
Added generic mapclick event to map view.

### CODE REVIEW

File: ios/Classes/TiMapView.m [hunk 1]
```diff
@@ -108,8 +108,23 @@ -(void)handleOverlayTap: (UIGestureRecognizer*)tap
...
+
+//TiProxy * mapProxy = [self proxy];
+//CGPoint location = [sender locationInView:map];
+//CLLocationCoordinate2D coord = [map convertPoint:location toCoordinateFromView:map];
+//NSNumber *lat = [NSNumber numberWithDouble:coord.latitude];
+//NSNumber *lng = [NSNumber numberWithDouble:coord.longitude];
+//NSDictionary * event = [NSDictionary dictionaryWithObjectsAndKeys:
+//                        mapProxy,@"map", lat,@"latitude", lng,@"longitude", nil];
+//if ([mapProxy _hasListeners:@"click"]) {
+//    [mapProxy fireEvent:@"click" withObject:event];
```
Comment (hansemannn) [inline ios/Classes/TiMapView.m]: Remove comments

File: ios/Classes/TiMapView.m [hunk 2]
```diff
@@ -1128,6 +1143,20 @@ -(void)handleLongPressOnMap:(UIGestureRecognizer *)sender
...
+
+    CLLocationCoordinate2D clickCoordinate = MKCoordinateForMapPoint(point);
+    NSNumber *lat = [NSNumber numberWithDouble:clickCoordinate.latitude];
+    NSNumber *lng = [NSNumber numberWithDouble:clickCoordinate.longitude];
+    NSDictionary * event = [NSDictionary dictionaryWithObjectsAndKeys:
+                            mapProxy,@"map", lat,@"latitude", lng,@"longitude", nil];
+    if ([mapProxy _hasListeners:@"mapclick"]) {
+        [mapProxy fireEvent:@"mapclick" withObject:event];
+    }
+}
```
Comment (hansemannn) [inline ios/Classes/TiMapView.m]: Fix indentation

Comment (bridgepoint) [inline ios/Classes/TiMapView.m]: sorry what am i missing here?

Comment (hansemannn) [inline ios/Classes/TiMapView.m]: bildschirmfoto 2017-03-06 um 11 49 33

Either use the indentation from 1148 or from the rest of the method. For new ones, the second one is preferred (4 spaces).

**Comment by hansemannn** [comment]
Hey @bridgepoint, I made the changes for you. PR **approved**, thank you!

**Pull request merged**